tests/cpptests/test_cpp_iterator_idlist.cpp

```diff
@@ -20,7 +20,14 @@ class IDListIteratorCommonTest : public ::testing::TestWithParam<std::vector<t_d
 
   void SetUp() override {
     docIds = GetParam();
-    iterator_base = IT_V2(NewIdListIterator)(docIds.data(), docIds.size(), 1.0);
+    // do a copy of the docIds vector and sort them before passing them to the iterator
+    std::sort(docIds.begin(), docIds.end(), cmp_docids);
```

Comment:
Each test sorts `docIds` again at the moment. Let's remove the sorting from the tests.

Reply:
why would we? if it is a prerequisite of the Iterator? Wdym?

Reply:
I meant keep the sorting in the `Setup` only, we don't need to sort the sorted list again in every test